src/js/typeahead/helpers.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 module.exports = {
-  sorter: function(input) {
-    return function sorter(a, b) {
+  sorter: function sorter(input) {
+    return function sort(a, b) {
```

Comment:
oh ignore me - we aren't transpiling are we :man_facepalming: we probably should be - we are using browserify anyway so its not like it adds much of an overhead